jenkins_pipelines/environments/manager-4.2-dev-acceptance-tests-NUE

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '20', daysToKeepStr: '4')),
+        disableConcurrentBuilds(),
+        pipelineTriggers([cron('H H/4 * * *')]),
+        parameters([
+            string(name: 'cucumber_gitrepo', defaultValue: 'https://github.com/SUSE/spacewalk.git', description: 'Testsuite Git Repository'),
+            string(name: 'cucumber_ref', defaultValue: 'Manager-4.2', description: 'Testsuite Git reference (branch, tag...)'),
```

Comment:
This seems correct for the default in the future but keep in mind that this doesn't exist yet. Maybe we make `master` the default for now until we create `Manager-4.2`?

Reply:
The jobs will be disabled until we have `Manager-4.2`. No need to run this until we branch :-)